e2e_tests/tests/command/command.py

```diff
@@ -29,9 +30,12 @@ def __init__(self, process: subprocess.Popen, detach: bool = False):
             while not m and iterations < max_iterations:
                 line = next(iterator)
                 iterations += 1
-                m = re.search(rb"Launched .* \(id: (.*)\)", line)
+                m = re.search(rb"Launched .* \(id: (.*), name: (.*)\)", line)
             assert m is not None
             self.task_id = m.group(1).decode() if m else None
+            self.task_name = m.group(2).decode() if m else None
+            assert self.task_id is not None, "Task ID could not be found"
+            assert self.task_name is not None, "Task name could not be found"
```

Comment:
why weren't we checking these before 🤔 can the assumption be incorrect somewhere?